modules/thermal_hydraulics/doc/content/source/postprocessors/NodalEnergyFluxPostprocessor.md

```diff
@@ -1,20 +1,14 @@
 # NodalEnergyFluxPostprocessor
 
-!alert construction title=Undocumented Class
-The NodalEnergyFluxPostprocessor has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /Postprocessors/NodalEnergyFluxPostprocessor
 
-## Overview
-
-!! Replace these lines with information regarding the NodalEnergyFluxPostprocessor object.
+The energy flux $\Phi$ through nodes is computed as:
 
-## Example Input File Syntax
+!equation
+\Phi = \sum_{nodes} \alpha \rho u A H
 
-!! Describe and include an example of how to use the NodalEnergyFluxPostprocessor object.
+where $\alpha$ is the phase fraction, $\rho$ is the phase density, $u$ is the phase velocity,
+$A$ the local area and $H$ the specific total enthalpy.
```

Comment:
```suggestion
$A$ the local area, and $H$ the specific total enthalpy.
```

Reply:
What's your position on the Oxford comma? No one seems to use it, but everyone seems to think it's the right thing to do. I prefer to be a hero.

Reply:
my English teachers never got on to me for this so I m fairly neutral

Reply:
I use it!  If you ever see me write that famous "We invited the strippers, JFK and Stalin" sentence, don't assume I forgot a comma; rush to a news website right away and see what dirt has just been declassified.